src/components/BorrowerProfile/LendCta.vue

```diff
@@ -635,6 +649,9 @@ export default {
 		},
 		unreservedAmount(newValue, previousValue) {
 			// set initial selected value for sub 25 loan if shown
+			if (this.enableFiveDollarsNotes && this.activeCookie !== '') {
```

Comment:
It looks like this `watch` doesn't have the same logic as `mounted`. Here `selectedOption` is only affected by one of the UTM campaign use cases, where `mounted` is affect by both. Maybe the logic in `mounted` needs to be extracted and used in both the `watch` and the `mounted`.

Reply:
this should be the same logic as LendCtaExp right? that watch function is cleaner

Reply:
It's hard to say without digging in. These two CTA components are being used in very different parent components.

Reply:
Okay it looks like the two CTAs actually have very similar logic, once just uses a separate method. We couldn't make the logic identical between the two CTAs without some pretty significant functional testing, but it's definitely still an option. Because like you said, the logic should be similar.